governance/parameter-docs/param-proportional-kick-incentive.md

```diff
@@ -9,24 +9,24 @@ Technical Docs:
 ```
 
 ## Description
-* What is this parameter?
-* How does it fit in with the protocol and surrounding contracts?
+The Proportional Kick Incentive parameter controls the percentage of the target amount of DAI to raise from bidders (debt + stability fees + liquidation penalty) that will be destined to incentivize keepers when liquidating a vault or resetting an already existing auction.
 
 ## Purpose
-* Why does this parameter exist?
+Governance may periodically increase the Debt Floor. When this happens, it is usually because gas has become so expensive that it impacts the efficiency of liquidations — that is, the cost of calling auction-related functions may exceed the value of the collateral to be liquidated. Incentives may be used as a remedy to this potential issue and possibly as a way for the protocol to keep the Debt Floor lower. The Proportional Kick Incentive ensures that auction initialization or reset is being compensated to cover keepers gas costs and spurs the prompt liquidation of unsafe vaults.
 
 ## Trade-offs
-* What dangers does this parameter represent?
-* What advantages does this parameter represent?
+Increasing the Proportional Kick Incentive parameter should make auctions trigger more quickly. Prompt liquidations prevent the accrual of bad debt. Increasing this parameter may also help in keeping the Debt Floor lower. A lower Debt Floor makes creating Vaults more universally accessible in terms of required capital. 
+
+On the other hand, increasing this parameter also carries the risk of prompting the proliferation of vaults created with the sole intent of reaping the incentive by liquidating them on purpose.
```

Comment:
Probably avoid proliferation, too wordy :) Bear in mind that the liquidation penalty directly counteracts this. So `chip` should be less than `chop` always.